src/Db/Dump/PKSimDB_dump.txt

```diff
@@ -5952,10 +5955,10 @@ INSERT INTO tab_calculation_method_rates VALUES('LinksCommon','MetabolizationSpe
 INSERT INTO tab_calculation_method_rates VALUES('LinksCommon','MetabolizationSpecific_MM','CP * kcat * V * K_water * C / (Km + K_water * C)','Amount per time');
 INSERT INTO tab_calculation_method_rates VALUES('LinksCommon','MucosaActiveEffluxToLumen_Hill','TM * kcat * (K_water_cell * max(C_cell;0))^alpha / (Km^alpha+(K_water_cell * max(C_cell;0))^alpha)','Amount per time');
 INSERT INTO tab_calculation_method_rates VALUES('LinksCommon','MucosaActiveEffluxToLumen_MM','TM * kcat * K_water_cell * C_cell / (Km+K_water_cell * C_cell)','Amount per time');
-INSERT INTO tab_calculation_method_rates VALUES('LinksCommon','MucosaActiveInfluxFromLumen_Hill','-TM * kcat * max(min(Solubility/MW; DrugLiquid/Liquid);0)^alpha / (Km^alpha+max(min(Solubility/MW; DrugLiquid/Liquid);0)^alpha)','Amount per time');
-INSERT INTO tab_calculation_method_rates VALUES('LinksCommon','MucosaActiveInfluxFromLumen_MM','-TM * kcat * min(Solubility/MW; DrugLiquid/Liquid) / (Km+min(Solubility/MW; DrugLiquid/Liquid))','Amount per time');
-INSERT INTO tab_calculation_method_rates VALUES('LinksCommon','MucosaPgPToLumen_Hill','min(TM * kcat * max(min(Solubility/MW; DrugLiquid/Liquid);0)^alpha / (Km^alpha+max(min(Solubility/MW; DrugLiquid/Liquid);0)^alpha); P_int_trans*Aeff*P_int_trans_lum_cell_factor * min(Solubility/MW; DrugLiquid/Liquid)) + TM * kcat * (K_water_cell * max(C_cell;0))^alpha / (Km^alpha+(K_water_cell * max(C_cell;0))^alpha)','Amount per time');
-INSERT INTO tab_calculation_method_rates VALUES('LinksCommon','MucosaPgPToLumen_MM','min(TM * kcat / (Km+min(Solubility/MW; DrugLiquid/Liquid)); P_int_trans*Aeff*P_int_trans_lum_cell_factor) * min(Solubility/MW; DrugLiquid/Liquid) + TM * kcat * K_water_cell * C_cell / (Km+K_water_cell * C_cell)','Amount per time');
+INSERT INTO tab_calculation_method_rates VALUES('LinksCommon','MucosaActiveInfluxFromLumen_Hill','-TM * kcat * EffectiveLumenConcentration^alpha / (Km^alpha+EffectiveLumenConcentration^alpha)','Amount per time');

```

Comment:
so here and in other formulas: use the effective concentration in lumen parameter instead of min(blabla)